Rakefile

```diff
@@ -2,7 +2,7 @@ require 'rake/testtask'
 
 # Ruby tests
 Rake::TestTask.new do |t|
-  t.pattern = '**/test_*.rb'
+  t.test_files = FileList['**/test_*.rb'].exclude('vendor/**/*')
```

Comment:
This really tripped me up...without this change, Rake was running all the tests for the vendored gems, and was failing.